python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -1089,6 +1091,12 @@ def _validate_partitions_existence(self) -> None:
     def resolve_hasAssetChecks(self, graphene_info: ResolveInfo) -> bool:
```

Comment:
we can probably remove this, along with the top level resolver. I could do that in a follow up